numpy/add_newdocs.py

```diff
@@ -463,6 +463,45 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Convenience function to create multiple nditers from a single ndarray.
+
+    Parameters (see nditer)
+    ---------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  sequence of sequences
```

Comment:
Let's go with `tuple of tuples`, since that's what the error messages in `nested_iters` ask for

Reply:
FWIW, the error message is misleading, the [code](https://github.com/numpy/numpy/blob/master/numpy/core/src/multiarray/nditer_pywrap.c#L898) can accept either tuple or list. Should I add a fix to the error message?

Reply:
There are a lot of places where we encourage users to pass tuples, but accept lists for backwards compat. I'd be inclined to keep this the same way.